metrics.md

```diff
@@ -3,44 +3,59 @@
 
 | Name | Metric type | Usage | Sample collection event |
-|---------------------------------|-------------|-------------------------------------------------------------------|-------------------------------------|
-| beaconchain_peers              | Gauge       | Tracks number of peers                                            | When a new peer is added or dropped |
-| beaconchain_current_slot | Gauge | Latest slot recorded by the beacon chain | Each time slot changes |
-| beaconchain_current_justified_epoch         | Gauge       | Current justified epoch                                           | On each epoch                       |
-| beaconchain_current_finalized_epoch         | Gauge       | Current finalized epoch                                           | On each epoch                       |
-| beaconchain_current_prev_justified_epoch    | Gauge       | Current previously justified epoch                                | On each epoch                       | 
-| beaconchain_current_epoch_live_validators   | Gauge       | Number of active validators who reported for the current epoch    | On each epoch                       |
-| beaconchain_previous_epoch_live_validators  | Gauge       | Number of active validators who reported for the previous epoch   | On each epoch
-| beaconchain_reorg_events_total              | Counter     | Occurrence of a reorganization of the chain                       | On fork choice                      |
-| beaconchain_pending_deposits                | Gauge       | Number of pending deposits                                        | Upon processing eth1 events watching the deposit contract |
-| beaconchain_pending_exits                | Gauge       | Number of pending exits                                        | On each epoch                              |
-| beaconchain_total_deposits                  | Gauge       | Number of total deposits                                          | Upon processing eth1 events watching the deposit contract |
-| beaconchain_previous_epoch_stale_blocks     | Gauge       | Number of blocks not included into canonical chain in the previous epoch                                                                                                                  | On each epoch                       |
-| beaconchain_propagated_attestations         | Gauge       | Number of distinct attestations to a slot received from the wire                                                                                                                 | When attestation inclusion delay passed     |
+|-----------------------------------------------|-------------|-------------------------------------------------------------|----------------------|
+| `beaconchain_peers`                           | Gauge       | Tracks number of peers                                      | On peer add/drop     |
```

Comment:
Minor nit: I think this should be `libp2p_peers` unless we're trying to track which peers are subscribed to the beacon chain topic.

Reply:
yup